spec/requests/case_contacts_spec.rb

```diff
@@ -53,6 +53,25 @@
             post case_contacts_url, params: {case_contact: valid_attributes}
           }.to change(CaseContact, :count).by(2)
         end
+
+        context "when the URL contains ?success=true" do
+          let!(:casa_case) { create(:casa_case, casa_org: organization) }
```

Comment:
```suggestion
          let(:casa_case) { create(:casa_case, casa_org: organization) }
```

You can let it lazy evaluate since it's going to run in the before block

Reply:
actually I think you don't need this one since a casa_case is already being created at the top L7